modules/apps/commerce/commerce-discount-service/src/main/resources/META-INF/portlet-model-hints.xml

```diff
@@ -88,6 +88,7 @@
 		<field name="commerceDiscountId" type="long" />
 		<field name="classNameId" type="long" />
 		<field name="classPK" type="long" />
+		<field name="discountTypeSettings" type="String" />
```

Comment:
@CrescenzoRegaLF add something like this https://github.com/liferay-commerce/liferay-portal/pull/4201/commits/05c3d154e3e231e4d80421e5c15fd691dbf177c1 because otherwise this https://liferay.atlassian.net/browse/COMMERCE-12255 will happen again for this feature